Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -288,7 +288,7 @@ namespace AZ::Reflection
                     path.append("/");
                     path.append(AZStd::string::format("%zu", parentData.m_childElementIndex));
                 }
-                else if (nodeData.m_classElement && nodeData.m_classElement->m_editData)
+                else if (nodeData.m_classElement)
```

Comment:
Will this re-introduce data elements that aren't exposed to the `EditContext` ? Or will that still be protected further down?